client/src/assets/images/svg/index.tsx

```diff
@@ -55,3 +57,9 @@ export const IconMenu = () => (
     <path d="M128.4 106.5c0 12.1-9.8 21.9-21.9 21.9H21.9C9.8 128.4 0 118.6 0 106.5V21.9C0 9.8 9.8 0 21.9 0h84.6c12.1 0 21.9 9.8 21.9 21.9v84.6zm180.9-84.6c0-12.1-9.8-21.9-21.9-21.9h-84.6c-12.1 0-21.9 9.8-21.9 21.9v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9V21.9zm180.9 0c0-12.1-9.8-21.9-21.9-21.9h-84.6c-12.1 0-21.9 9.8-21.9 21.9v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9V21.9zM309.3 202.8c0-12.1-9.8-21.9-21.9-21.9h-84.6c-12.1 0-21.9 9.8-21.9 21.9v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9v-84.6zm180.9 0c0-12.1-9.8-21.9-21.9-21.9h-84.6c-12.1 0-21.9 9.8-21.9 21.9v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9v-84.6zm-361.8 0c0-12.1-9.8-21.9-21.9-21.9H21.9C9.8 180.9 0 190.7 0 202.8v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9v-84.6zm159 159h-84.6c-12.1 0-21.9 9.8-21.9 21.9v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9v-84.6c0-12.1-9.8-21.9-21.9-21.9zm96.3 128.4h84.6c12.1 0 21.9-9.8 21.9-21.9v-84.6c0-12.1-9.8-21.9-21.9-21.9h-84.6c-12.1 0-21.9 9.8-21.9 21.9v84.6c0 12.1 9.8 21.9 21.9 21.9zM106.5 361.8H21.9C9.8 361.8 0 371.6 0 383.7v84.6c0 12.1 9.8 21.9 21.9 21.9h84.6c12.1 0 21.9-9.8 21.9-21.9v-84.6c0-12.1-9.8-21.9-21.9-21.9z" />
   </svg>
 );
+
+export const Wave = ({ reversed }: { reversed: boolean }) => (
```

Comment:
please create interface for `{ reversed: boolean }` and add type definitions for components/functions